<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowTableView</class>
 <widget class="QWidget" name="ShowTableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShowTableView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="sortByColumnQGroupBox">
     <property name="title">
      <string>Sortuj po nazwie kolumny</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="columnNameQComboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="sortQPushButton">
        <property name="text">
         <string>Sortuj</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="ascendingQRadioButton">
        <property name="text">
         <string>Rosnąco</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="descendingQRadioButton">
        <property name="text">
         <string>Malejąco</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableQTableWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
